my-react-app/src/app/screens/Login/components/Fields/customInput.js

```diff
@@ -0,0 +1,12 @@
+import React, { Fragment } from 'react';
+
+export const CustomInput = props => {
```

Comment:
Use `export default` at the end of the file for components like this, where it´s the only export in the file